pfcpiface/conn.go

```diff
@@ -65,6 +65,25 @@ type PFCPConn struct {
 	hbCtxCancel context.CancelFunc
 
 	pendingReqs sync.Map
+	pConnId     uint32
+}
+
+func (pConn *PFCPConn) GenerateFseid() uint64 {
```

Comment:
Could you please add a comment explaining what is the structure of a new F-SEID format and how it is generated?